pyannote/audio/utils/metric.py

```diff
@@ -104,25 +104,37 @@ def metric_components(cls):
         return ["total", "false alarm", "missed detection", "confusion"]
 
     def compute_components(
-        self, reference, hypothesis, uem: Optional[Timeline] = None,
+        self,
+        reference,
+        hypothesis,
+        uem: Optional[Timeline] = None,
     ):
-        return self.compute_components_helper(hypothesis, reference)
+        return self.compute_components_helper(hypothesis, reference, uem)
 
     @singledispatchmethod
-    def compute_components_helper(self, hypothesis, reference):
+    def compute_components_helper(self, hypothesis, reference, uem=None):
         klass = hypothesis.__class__.__name__
         raise NotImplementedError(
             f"Providing hypothesis as {klass} instances is not supported."
         )
 
     @compute_components_helper.register
-    def der_from_ndarray(self, hypothesis: np.ndarray, reference: np.ndarray, **kwargs):
+    def der_from_ndarray(
+        self,
+        hypothesis: np.ndarray,
+        reference: np.ndarray,
+        uem: Optional[Timeline] = None,
+        **kwargs,
```

Comment:
```suggestion
        uem: Optional[Timeline] = None,
```

Reply:
I think `**kwargs` was here because I anticipated this PR. It is therefore no longer needed.